wrims-comparison-test/src/test/java/wrims/compare/stepdefinitions/AzureStepDefinitions.java

```diff
@@ -0,0 +1,81 @@
+package wrims.compare.stepdefinitions;
+
+import io.cucumber.java.en.Given;
+import io.cucumber.java.en.Then;
+import io.cucumber.java.en.When;
+import wrims.compare.utils.AzureUtils;
+import wrims.compare.utils.ComputeTestUtils;
+import wrims.compare.utils.LocalFileUtils;
+
+import java.io.InputStream;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.StandardCopyOption;
+
+public class AzureStepDefinitions {
```

Comment:
Given this is running in Azure and can have environment issues occur, This class should have logging/observability added to it in order to assist with debugging should an error occur.